client/components/telecom/telephony/group/number/feature/ovhPabx/menu/telephony-group-number-feature-ovh-pabx-menu.component.controller.js

```diff
@@ -145,7 +145,7 @@ angular.module('managerApp').controller('telephonyNumberOvhPabxMenuCtrl', functi
         if (error.status === 403) {
           errorTranslationKey = 'telephony_number_feature_ovh_pabx_menu_action_delete_error_used';
         }
-        TucToast.error([$translate.instant(errorTranslationKey), _.get(error, 'data.message') || ''].join(' '));
+        TucToast.error($translate.instant(errorTranslationKey));
```

Comment:
Not yours but in addition to that change you can also removed the following:

`return $q.reject(error);` (and remove `$q` injection if no more used) because no catch is made when `onMenuDeleteConfirmBtnClick` method is called.

Is that make sense to you?

Reply:
Yes sure :)